sdk/liftrqumulo/Azure.ResourceManager.LiftrQumulo/api/Azure.ResourceManager.LiftrQumulo.netstandard2.0.cs

```diff
@@ -0,0 +1,480 @@
+        public virtual System.Threading.Tasks.Task<Azure.Response<Azure.ResourceManager.LiftrQumulo.EndpointResource>> GetEndpointAsync(string endpointName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.ResourceManager.LiftrQumulo.EndpointCollection GetEndpoints() { throw null; }
+        public virtual Azure.Response<Azure.ResourceManager.LiftrQumulo.ProjectResource> GetProject(string projectName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual System.Threading.Tasks.Task<Azure.Response<Azure.ResourceManager.LiftrQumulo.ProjectResource>> GetProjectAsync(string projectName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.ResourceManager.LiftrQumulo.ProjectCollection GetProjects() { throw null; }
+        public virtual Azure.Response<Azure.ResourceManager.LiftrQumulo.StorageMoverResource> RemoveTag(string key, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual System.Threading.Tasks.Task<Azure.Response<Azure.ResourceManager.LiftrQumulo.StorageMoverResource>> RemoveTagAsync(string key, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.Response<Azure.ResourceManager.LiftrQumulo.StorageMoverResource> SetTags(System.Collections.Generic.IDictionary<string, string> tags, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual System.Threading.Tasks.Task<Azure.Response<Azure.ResourceManager.LiftrQumulo.StorageMoverResource>> SetTagsAsync(System.Collections.Generic.IDictionary<string, string> tags, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.Response<Azure.ResourceManager.LiftrQumulo.StorageMoverResource> Update(Azure.ResourceManager.LiftrQumulo.Models.StorageMoverPatch patch, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual System.Threading.Tasks.Task<Azure.Response<Azure.ResourceManager.LiftrQumulo.StorageMoverResource>> UpdateAsync(Azure.ResourceManager.LiftrQumulo.Models.StorageMoverPatch patch, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+    }
+}
+namespace Azure.ResourceManager.LiftrQumulo.Models
+{
+    public partial class AgentPatch
```

Comment:
Since the resource model has prefix, all related models should have the prefix too.